src/sentry/integrations/github/integration.py

```diff
@@ -86,6 +86,20 @@ class GitHubIntegration(IntegrationInstallation, GitHubIssueBasic, RepositoryMix
     def get_client(self):
         return GitHubAppsClient(integration=self.model)
 
+    def get_codeowner_file(self, repo, ref=None):
```

Comment:
@MeredithAnya unnecessary arg. `ref` is not used in this function.

Reply:
@NisanthanNanthakumar so I added: https://github.com/getsentry/sentry/pull/24998/files#diff-2eff1b915a66947e07be932f69aead405b923a4de74873b8d38a5da027b8708aR102, I think it might be easier to keep the arguments the same instead of changing the signature between providers. 

I guess I could have done the following if I wanted to for the base, but I think its okay to have the ref unused for now. It's possible we may want to use the ref later if we want to compare their files and let them know something is out of date
```python
def get_codeowner_file(self, repo, *args, **kwargs):
    raise NotImplementedError
```